speechbrain/alignment/ctc_segmentation.py

```diff
@@ -0,0 +1,652 @@
+#!/usr/bin/env python3
+# 2021, Technische Universität München, Ludwig Kürzinger
+"""Perform CTC segmentation to align utterances within audio files."""
```

Comment:
Mention here that one package needs to be installed.

Reply:
OK - will be included in the next commit